### PR TITLE
chore(lint): Fixed warnings on the tsdb/engine/wal subpackage

### PR DESCRIPTION
**Pull request by marcosinger**
Hi there, this is my first PR, so please let me know if I did something wrong :)

- [X] CHANGELOG.md updated
- [X] Rebased/mergable
- [X] Tests pass
- [X] Sign [CLA](http://influxdb.com/community/cla.html)

Issue #4098

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 With this release InfluxDB is moving to Go 1.5.
 
 ### Features
+- [#4098](https://github.com/influxdb/influxdb/issues/4098): Fixed lint warnings on the tsdb/engine/wal subpackage
```
Comment (otoolep) [inline CHANGELOG.md]: Can you move this to the bug fix section?

Comment (marcosinger) [inline CHANGELOG.md]: sure thing

**Comment by marcosinger** [comment]
done @otoolep

**Comment by otoolep** [comment]
Thanks @marcosinger -- we're performing a major refactor of this code right now, so we'll probably hold off on merging this for now.

**Comment by marcosinger** [comment]
roger @otoolep .. I'd like to help with more PR's in a near future, so I'll take a look on the issue's list to see something to tackle in the meantime.. should I close it?

**Comment by otoolep** [comment]
Yeah @marcosinger -- closing is probably best right now, and then revisit.

**Pull request closed without merging**